includes/class-wc-calypso-bridge-admin-setup-checklist.php

```diff
@@ -86,12 +86,25 @@ private function handle_redirects() {
 	}
 
 	/**
-	 * Returns if the checklist is "done" via the user clicking the "I'm done" button.
+	 * Returns if the checklist is "done" via the user clicking the "I'm done" button
+	 * or if the store already contains any orders.
 	 *
-	 * @return bool True if the done button has been clicked.
+	 * @return bool
 	 */
 	private function is_checklist_done() {
-		return (bool) get_option( 'atomic-ecommerce-setup-checklist-complete', false );
+		if ( apply_filters( 'wc_calypso_bridge_development_mode', false ) ) {
+			return false;
+		}
+
+		global $wpdb;
+		$order_count = (int) $wpdb->get_var(
```

Comment:
In lieu of a custom query, did you consider using `WC_Order_Query` or `wp_count_posts()`

Reply:
Because this is run so early, `WC_Order_Query` is not yet available and `wp_count_posts` does not seem to return the correct amounts (and because it's grouped by status, we'd have to loop through all default/custom statuses to check if any existed).